### PR TITLE
System Metrics - prod node GA Tracking ID

### PR DESCRIPTION
**Pull request by janciesla8818**
This PR:
* enables system metrics service by default on all nodes
* new GA Tracking ID is added for prod nodes
* metrics tests refactoring

### CODE REVIEW

**Comment by mongoose-im** [comment]
[7835.1](https://travis-ci.org/esl/MongooseIM/jobs/642403258) / Erlang 22.0 / small_tests / 310cb811297d50160512ddf130d9c9999eba55b8
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2610/7835/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2610/7835/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-27_14.33.47/lib.mongooseim.logs/run.2020-01-27_14.33.48/suite.log.html)

File: big_tests/tests/service_mongoose_system_metrics_SUITE.erl [hunk 1]
```diff
@@ -412,15 +414,22 @@ configure_additional_tracking_id(Node) ->
     {atomic, ok} = mongoose_helper:successful_rpc(Node, ejabberd_config, add_local_option, TrackingIdArgs).
 
 remove_additional_tracking_id(Node) ->
-    mongoose_helper:successful_rpc(Node, ejabberd_config, del_local_option, [ extra_google_analytics_tracking_id ]).
+    mongoose_helper:successful_rpc(
+        Node, ejabberd_config, del_local_option, [ extra_google_analytics_tracking_id ]).
+
+    remove_service_from_config(Service) ->
```
Comment (michalwski) [inline big_tests/tests/service_mongoose_system_metrics_SUITE.erl]: Unnecessary indentation:
```suggestion
remove_service_from_config(Service) ->
```

Comment (michalwski) [reply big_tests/tests/service_mongoose_system_metrics_SUITE.erl]: And in other lines in the function.

**Comment by mongoose-im** [comment]
[7836.1](https://travis-ci.org/esl/MongooseIM/jobs/642425057) / Erlang 22.0 / small_tests / 9b02d3a16282c99c370ef87747d22cfc21abca9b
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2610/7836/small_tests.22.0) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2610/7836/small_tests.22.0/small/ct_run.mongooseim@localhost.2020-01-27_15.37.25/lib.mongooseim.logs/run.2020-01-27_15.37.26/suite.log.html)

**Pull request merged**